.github/ISSUE_TEMPLATE/Feature_Request_Windows_Customization.yml

```diff
@@ -0,0 +1,71 @@
+name: "Suggest a new setting/feature within Windows Customization"
+description: Suggest a new feature or improvement within Windows Customization (this does not mean you have to implement
+  it)
+labels:
+- Issue-Feature
+- Area-Windows-Customization
+body:
+- id: minwindowsversion
+  type: input
+  attributes:
+    label: Minimum Supported Windows Version
+    placeholder: "10.0.23400.0"
+    description: What is the minimum version of Windows that should support this feature?
+  validations:
+    required: true
+
+- id: maxwindowsversion 
+  type: input 
+  attributes: 
+    label: Maximum Supported Windows Version 
+    placeholder: "10.0.23400.0" 
+    description: What is the maximum version of Windows that should support this feature? If there is no maximum, please enter "None". 
+  validations:
+    required: true
+
+- id: registrykeys
+  type: textarea
+  attributes:
+    label: Registry Keys
+    placeholder: What registry keys are needed to enable this feature? Please provide the full path and value. If not applicable, please enter "None".
+  validations:
+    required: true 
+
+- id: ispublic
+  type: dropdown
```

Comment:
What is the purpose of asking for documentation?

Reply:
We need to confirm that the behavior is publicly documented before we approve the PR. There may be requests to enable specific behaviors that are well known, but not publicly documented and we'd like to ensure that we are not erroneously assuming some behavior that works now will work forever if it is not publicly documented and available by Microsoft.

Reply:
I see.  You're equating public documentation to supported behavior.  Searching for public documentation is our job.  Other developers should not have to scour the Microsoft docs to figure out if documentation exists.

Additionally, without your comment, I would ask myself "Why do I need to find if this has any public documentation?"  I bet many other developer will ask the same thing too.